src/app/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-ubs-admin-order-details-form',
+  templateUrl: './ubs-admin-order-details-form.component.html',
+  styleUrls: ['./ubs-admin-order-details-form.component.scss']
+})
+export class UbsAdminOrderDetailsFormComponent {
```

Comment:
Shouldn't you configure any `formGroup` for Order Details in the scope of that task?

Reply:
Added